js/ui/viewSelector.js

```diff
@@ -126,26 +128,116 @@ const ViewsDisplayLayout = new Lang.Class({
     Signals: { 'allocated-size-changed': { param_types: [GObject.TYPE_INT,
                                                          GObject.TYPE_INT] } },
 
-    _init: function(appDisplayActor) {
+    _init: function(entry, appDisplayActor, searchResultsActor) {
         this.parent();
 
+        this._entry = entry;
         this._appDisplayActor = appDisplayActor;
+        this._searchResultsActor = searchResultsActor;
+
+        this._entry.connect('style-changed', Lang.bind(this, this._onStyleChanged));
         this._appDisplayActor.connect('style-changed', Lang.bind(this, this._onStyleChanged));
+
+        this._heightAboveEntry = 0;
+        this.searchResultsTween = 0;
```

Comment:
I honestly haven't thought much about it, this is pretty much carried over from eos-desktop :-)

I assume it's fine, though, knowing it's being working fine for a while?